lib/server.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const net = require('net');
+const EE = require('events');
+const Client = require('../module/client.js');
+const PORT = process.env.PORT || 3000;
+const pool =[];
+const server = net.createServer();
+const ee = new EE();
+
+ee.on('\\nick', function(client, string){
+  client.nickname = string;
+});
+
+ee.on('\\all', function(client, string){
+  pool.forEach( user => {
+    user.socket.write(`${client.nickname}: ` + string);
```

Comment:
@loganlsr I know this came from Duncan's demo but, to keep things consistent and progressive, try using string interpolation as opposed to strict concatenation